code/game/objects/items/hot_potato.dm

```diff
@@ -52,6 +52,9 @@
 	var/atom/location = loc
 	location.visible_message(span_userdanger("[src] [detonate_explosion? "explodes" : "activates"]!"), span_userdanger("[src] activates! You've ran out of time!"))
 	if(detonate_explosion)
+		var/mob/living/victim_mob = loc
+		if((src in victim_mob.held_items) && istype(victim_mob, /mob/living))
+			victim_mob.gib(DROP_ALL_REMAINS)
```

Comment:
```suggestion
	if(detonate_explosion && isliving(loc))
		var/mob/living/victim_mob = loc
		if(get_held_index_of_item(src))
			victim_mob.gib(DROP_ALL_REMAINS)
```

Reply:
This but use `is_holding(src)` instead of `get_held_index_of_item(src)`